test/TensorFlowRuntime/tensor_group.swift

```diff
@@ -0,0 +1,72 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-build-swift -swift-version 5 -g %s -o %t/a.out
+// RUN: %target-codesign %t/a.out
+// RUN: %target-run %t/a.out
+// REQUIRES: executable_test
+//
+// `TensorGroup` tests.
+
+import TensorFlow
+import StdlibUnittest
+
+var TensorGroupTests = TestSuite("TensorGroup")
+
+extension TensorDataType : Equatable {
+  public static func == (lhs: TensorDataType, rhs: TensorDataType) -> Bool {
+    return Int(lhs._cDataType.rawValue) == Int(rhs._cDataType.rawValue)
+  }
+}
+
```

Comment:
Could you please add an empty struct test?
`struct Empty : TensorGroup {}`

Reply:
Would that consist of simply adding this empty struct and making sure it compiles, or also checking for whether the data types and shapes are empty arrays?

Reply:
I added a simple test just in case and addressed all the feedback. :)